docs/troubleshooting_guide.md

```diff
@@ -0,0 +1,358 @@
+### Troubleshooting Guide:
+This guide provides a searchable list of errors for a possible match/solution.
+
+
+#### 1. Missing `dg_name` in `cluster_config.json` manifestation:
```

Comment:
Please don't include numbers;  markdown down't support auto-numebrins inside titles.  s/manifestation/file   And as with the preivous comment, trailing colon can be removed, and #### is probably too deep for a top-level question.

Reply:
Done